std_msgs/Header header
float64[3] z_int
float64[3] F_comp
int32[3] z_int_dir
int32[3] set_F_comp
float64[3] ki
float64[3] kc
int32[3] damp